tests/test_mathematical.py

```diff
@@ -2975,20 +2976,96 @@ def test_matmul_strided_vec_mat(self, shape, incx, incy, transpose):
         assert result is out
         assert_dtype_allclose(result, expected)
 
+    @pytest.mark.parametrize(
+        "order_pair",
```

Comment:
what if to define order variables here?
```suggestion
        "order1, order2, out_order",
```
and then below:
```python
def test_matmul_out1(self, order1, order2, out_order, dtype):
```